jest.config.js

```diff
@@ -1,7 +1,8 @@
 module.exports = {
   verbose: true,
   collectCoverage: true,
-  collectCoverageFrom: ['scripts/**/*.js'],
```

Comment:
Why did you remove this?

Reply:
We don't need that jest will only include files that are directly imported in our test files for coverage.

Reply:
That's the case. Jest is now taking `fixture` files also into coverage, and that's wrong.

Reply:
I will update it for excluding fixture files